src/containers/HomePage.js

```diff
@@ -1,43 +1,53 @@
 import React, { PureComponent } from 'react';
 import { connect } from 'react-redux';
+import { array, func } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
 import Menu from 'components/common/Menu';
 import LogoutButton from 'components/user/LogoutButton';
 import SimpleMap from 'components/common/maps/Map';
 import Welcome from 'components/user/Welcome';
 import CreateTargetForm from 'components/user/CreateTargetForm';
-import { array, func } from 'prop-types';
-
 import { loadTargets, addTarget } from '../actions/targetActions';
 import { loadTopics } from '../actions/topicActions';
-import { FormattedMessage } from 'react-intl';
+
+// Styles
```

Comment:
No need to have these comments, they tend to make files longer and it's pretty clear you're importing styles.